api/python/quilt3/packages.py

```diff
@@ -782,6 +919,10 @@ def build(self, name=None, registry=None, message=None):
         Returns:
             The top hash as a string.
         """
+        for _, entry in self.walk():
```

Comment:
After discussion about what a package is, I am changing this so that we support serialization on build()